validation.md

```diff
@@ -2324,99 +2323,102 @@ Email::defaults(function () {
 
 ### Use Cases & Examples
```

Comment:
This might alternatively be a table:
| Method | Description | Use case | example |
| ------ | ----------- | -------- | ------- |
| `rfcCompliant(strict: bool)` | Validates the email according to [RFC 5322](https://datatracker.ietf.org/doc/html/rfc5322).<br>Passing `true` enforces stricter RFC checks (e.g., rejects `invalid.@example.com`). | Use `rfcCompliant()` if you need standard RFC validation while allowing some unusual formats.<br>Use `rfcCompliant(true)` if you want to reject less typical but technically valid addresses. | **Basic RFC validation**<br>`Rule::email()->rfcCompliant();`<br>**Strict RFC validation**<br>`Rule::email()->rfcCompliant(true);` |

…